circuits/cpp/src/aztec3/circuits/kernel/private/native_private_kernel_circuit.cpp

```diff
@@ -256,13 +257,26 @@ void update_end_values(DummyComposer& composer,
     //     const auto& new_l2_to_l1_msgs = private_call_public_inputs.new_l2_to_l1_msgs;
     //     std::array<CT::fr, NEW_L2_TO_L1_MSGS_LENGTH> l1_call_stack;
 
-    //     for (size_t i = 0; i < new_l2_to_l1_msgs.size(); ++i) {
-    //         l1_call_stack[i] = CT::fr::conditional_assign(
-    //             new_l2_to_l1_msgs[i] == 0,
-    //             0,
-    //             CT::compress({ portal_contract_address, new_l2_to_l1_msgs[i] }, GeneratorIndex::L2_TO_L1_MSG));
-    //     }
-    // }
+    {
+        const auto& portal_contract_address = private_inputs.private_call.portal_contract_address;
```

Comment:
Can a copy of this line commented out a few lines above be removed now?

Reply:
Oh, ye will get rid of it.